### PR TITLE
fix a translation error and a format problem

### PR DESCRIPTION
**Pull request by NeoTse**

### CODE REVIEW

File: content/zh/docs/reference/glossary/cluster.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ tags:
 - operation
 ---
 
-<!-- ---
+<!--
```
Comment (tanjunchen) [inline content/zh/docs/reference/glossary/cluster.md:15]: @NeoTse 
see https://identity.linuxfoundation.org/projects/cncf
```
If you haven't yet, post a comment ("I signed it") to the PR—this is needed for the system to recheck your contribution authorization.
```

**Comment by NeoTse** [comment]
I signed it

**Pull request merged**